### DISCUSSION TITLE
What should I do if the hard disk temperature is not displayed

### DISCUSSION THREAD

**Discussion by chaomarks** [nicolargo/glances#2744] [General]
As mentioned, my device is arm architecture, and glances runs in docker, but it cannot obtain the hard disk temperature
![Uploading im
![0](https://github.com/nicolargo/glances/assets/93130581/280907c9-3c56-4541-94d6-e5faf5083fdd)

**Comment by nicolargo** [comment]
Currently, Glances v3 uses HDDTemp to get the hard disk temperature.

Do you have HDDTemp installed and working on your system ?
